src/main/kotlin/nl/info/zac/app/zaak/converter/RestZaakConverter.kt

```diff
@@ -134,6 +134,13 @@ class RestZaakConverter @Inject constructor(
             groep = groep,
             behandelaar = behandelaar,
             initiatorIdentificatie = initiator?.identificatienummer,
+            kvkNummer = when (initiator?.betrokkeneType) {
+                NIET_NATUURLIJK_PERSOON -> {
+                    val identificatie = initiator.betrokkeneIdentificatie as? NietNatuurlijkPersoonIdentificatie
+                    if (identificatie?.vestigingsNummer?.isNotBlank() == true) identificatie?.kvkNummer else null
```

Comment:
This will change in a future story where we will also add support for 'organisations with only a KVK nummer (and no vestigingsnummer)' but good not to implement that right away so this is just fine now. :-)